src/common/Cargo.toml

```diff
@@ -30,7 +30,7 @@ chrono = { version = "0.4", default-features = false, features = [
     "clock",
     "std",
 ] }
-chrono-tz = { version = "0.10", features = ["case-insensitive"] }
+chrono-tz = { version = "0.9", features = ["case-insensitive"] }
```

Comment:
There is a better workaround than downgrading the version. I am going to include it in a version bump PR soon.

https://gist.github.com/xiangjinwu/65c310adf84c381c17d99cb4439e1908

Reply:
#21860